extensions/2.0/Vendor/FB_materials_lightmap/schema/material.extension.FB_materials_lightmap.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "FB_materials_lightmap material extension",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "glTFProperty.schema.json"
+        }
+    ],
+    "properties": {
+        "lightmapTexture": {
+            "description": "The texture information for the lightmap.",
+            "allOf": [
+                {
+                    "$ref": "lightmap.schema.json"
```

Comment:
Should this be `textureInfo.schema.json` instead? Without that, other extensions like `KHR_texture_transform` would be considered invalid here.

Reply:
I'm still new to this, so I'm at your service. Can you point me to any documentation on how standardizing this part of the schema?

Reply:
Changed to `textureInfo.schema.json`